openapi.json

```
@@ -26,7 +26,7 @@
           {
             "name": "format",
             "in": "query",
-            "description": "Maximum number of records is 1000 i.e. 6000 rows for CSV downloads. The 'offset' parameter can be used for pagination with both formats.",
+            "description": "Ignored when 'Accept' header is present and valid.  \nMaximum number of records is 1000 i.e. 6000 rows for CSV downloads.  \nThe 'offset' parameter can be used for pagination with both formats.",
```

```suggestion
            "description": "Used as a fallback when the 'Accept' header is missing or specifies an unsupported media type.  \nMaximum number of records is 1000 i.e. 6000 rows for CSV downloads.  \nThe 'offset' parameter can be used for pagination with both formats.",
```

In my opinion, ROS OCP supports both JSON and CSV formats, but that doesn't mean other media types are invalid. As per [MDN's list of common MIME types](https://developer.mozilla.org/en-US/docs/Web/HTTP/Guides/MIME_types/Common_types), formats like text/css are valid but it is not applicable to endpoint of  ROS OCP service. It would be more accurate to refer to them as "unsupported" rather than "invalid."

Thoughts?